exo/topology/device_capabilities.py

```diff
@@ -198,12 +198,19 @@ def linux_device_capabilities() -> DeviceCapabilities:
       flops=CHIP_FLOPS.get(gpu_name, DeviceFlops(fp32=0, fp16=0, int8=0)),
     )
   elif Device.DEFAULT == "AMD":
-    # TODO AMD support
+    import pyamdgpuinfo
+
+    gpu_raw_info = pyamdgpuinfo.get_gpu(0)
+    gpu_name = gpu_raw_info.name
```

Comment:
On my NixOS box there's no `/usr/share/libdrm/amdgpu.ids`, so this value is assigned `None`
You might want to consider using this instead (for cases where that happens or there's no matching card):
```python
gpu_name = gpu_raw_info.name if gpu_raw_info.name is not None else "Unknown AMD"
```